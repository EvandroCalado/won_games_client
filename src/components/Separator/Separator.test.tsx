import { render } from '@testing-library/react';
import { Separator } from '.';

describe('Separator', () => {
  it('should render a snapshot', () => {
    const { container } = render(<Separator />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
