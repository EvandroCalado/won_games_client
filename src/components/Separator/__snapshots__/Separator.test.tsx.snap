// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator should render a snapshot 1`] = `
<div
  class="mx-auto max-w-7xl px-4 py-8"
>
  <hr
    class="mt-2 text-white/20"
  />
</div>
`;
